src/filesystem.cpp

```diff
@@ -717,7 +717,7 @@ bfs::path apple_userdata(const std::string& newprefdir)
 {
 	bfs::path dir;
 	std::string temp = newprefdir;
-	bool relative_ok = false;
+	const char* home_str = getenv("HOME");
```

Comment:
Note that this will break on Windows, where the variable is instead called `USERPROFILE`. (Or maybe it's lowercase? I'm not entirely sure if it matters.)